The maximum version of setuptools_scm is pinned below 6 for no apparent reason
and Void ships 7.x; remove the upper version restriction and also drop the
requirement for setuptools_scm_git_archive that was originally patched out in
the template.

--- a/setup.py
+++ b/setup.py
@@ -316,8 +316,7 @@
     setup_requires=[
         "certifi>=2020.06.20",
         "numpy>=1.17",
-        "setuptools_scm>=4,<7",
-        "setuptools_scm_git_archive",
+        "setuptools_scm>=4",
     ],
     install_requires=[
         "cycler>=0.10",
@@ -330,7 +329,7 @@
         "python-dateutil>=2.7",
     ] + (
         # Installing from a git checkout that is not producing a wheel.
-        ["setuptools_scm>=4,<7"] if (
+        ["setuptools_scm>=4"] if (
             Path(__file__).with_name(".git").exists() and
             os.environ.get("CIBUILDWHEEL", "0") != "1"
         ) else []
